### PR TITLE
fl - I changed the title of the ClassRoom search page to Search By Cl…

### PR DESCRIPTION
**Pull request by finnderman1**
…assroom. I also changed the description to 'Search based on your classroom pref'

### CODE REVIEW

**Comment by ryan8xia** [comment]
Why has localhost.sample been removed?

**Comment by finnderman1** [comment]
Because we dont need it anymore. We have the real localhost. I can add it back and then re commit if you would like

**Comment by ryan8xia** [comment]
Please leave the sample json in the repo

File: src/main/resources/templates/csdept/classroom/fragments/title.html [hunk 1]
```diff
@@ -1,3 +1,3 @@
-<h1>Search For Courses in CS Department Controlled Classrooms</h1>
-<p>This page allows you to search for all courses offered in CS department controlled classrooms in a specific quarter.
+<h1>Search by Classrooms</h1>
```
Comment (andrewhlu) [inline src/main/resources/templates/csdept/classroom/fragments/title.html]: Capitalize "By"?

Comment (pconrad) [reply src/main/resources/templates/csdept/classroom/fragments/title.html]: Some folks prefer lowercase.   But the most important thing is consistency.

**Comment by finnderman1** [comment]
I will resubmit without the sample JSON delete tomorrow in CSIL. Sorry!

**Pull request closed without merging**